css/properties/animation-timeline.json

```diff
@@ -40,6 +40,49 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "scroll": {
+          "__compat": {
+            "description": "<code>scroll()</code>",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/animation-timeline/scroll",
+            "spec_url": "https://w3c.github.io/csswg-drafts/scroll-animations-1/#funcdef-scroll",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "101",
+                "notes": "Zero scroll range is treated as 100% but should be 0% (see <a href='https://github.com/w3c/csswg-drafts/issues/7401'>csswg-drafts#7401</a>).",
```

Comment:
@queengooborg FYI, this note comes from discussion at https://bugzilla.mozilla.org/show_bug.cgi?id=1737918#c16

Essentially if an element is the source of the scroll timeline, and it does not have a scroll bar in the axis you specify, then the timeline has zero scroll range and is treated as inactive. The spec says that this should indicate 0 progress, but FF currently indicates 100 progress. 

I added the note to record the spec incompatibility, but it still isn't clear whether FF plans to revert or Chrome does something different or whatever. So for now, just noting the spec difference. When the FF engineer gets back from holiday sometime next month might update this.